src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py

```diff
@@ -285,7 +285,7 @@ def prepare_public_network(public_access, yes):
 
     if str(public_access).lower() == 'all':
         start_ip, end_ip = '0.0.0.0', '255.255.255.255'
-    elif str(public_access).lower() == 'none':
+    elif str(public_access).lower() == 'none' or str(public_access) == 'Disabled' or str(public_access) == 'Enabled':
```

Comment:
Why `str(public_access) == 'Enable'` also need to set `start_ip, end_ip = -1, -1`?